vuejs/src/components/board/BoardWrite.vue

```diff
@@ -3,18 +3,18 @@
         <h4>BOARD-WRITE</h4>
 
         <div class="poster-write">
-            <b-form>
+            <b-form @submit.prevent="boardSubmit" method="post">
```

Comment:
`@submit="boardSubmit"` 로 작업을 진행했을 때,
1. HTML `form submission canceled because the form is not connected vue` 경고 발생
2. 게시글 등록 후 목록으로 리다이렉트될 때, 작성한 글이 보이지 않음.
(로그를 보면 게시글 등록전에 목록이 먼저 불러와져서 그렇게 보이는 것)
위의 두가지 문제가 발생했다. 그래서 `@submit.prevent="boardSubmit"` 으로 변경했더니 문제 해결!